### ISSUE TITLE
new tab functionality breaks

### ISSUE THREAD

**Issue by gairadzi** [vimperator/vimperator-labs#684]
##### Issue type:

- Version compatibility

##### Version:

```
Vimperator:	3.15.0 (created: 2016/10/05 01:11:26)
Firefox:	Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:51.0) Gecko/20100101 Firefox/51.0
```

##### Description:

tabopen doesn't work.

##### Expected behavior:

a new tab to be opened with the address entered.

##### Steps to reproduce:

tabopen www.google.com

**Comment by fakedrake** [comment]
This has been discussed [quite](https://github.com/vimperator/vimperator-labs/issues/681) [a](https://github.com/vimperator/vimperator-labs/issues/671) [bit](https://github.com/vimperator/vimperator-labs/issues/674) and I understand there is a (flimsy?) solution in HEAD but why has it not made it to the firefox addon repo?

Btw I myself have started using conkeror which can't support devtools (easily) and looks like it won't for a while so I am still waiting for vimperator to start working again.

**Comment by timss** [comment]
Thanks for reporting this issue, but this seems to be a duplicate of #568 and was fixed in #675, ultimately released as [Vimperator 3.16](https://github.com/vimperator/vimperator-labs/releases/tag/vimperator-3.16.0). New version coming on [AMO](https://addons.mozilla.org/en-US/firefox/addon/vimperator/) soon™.

**Comment by timss** [comment]
@fakedrake The why is still, e10s or not https://github.com/vimperator/vimperator-labs/issues/211#issuecomment-249181301. Also, https://github.com/vimperator/vimperator-labs/issues/568#issuecomment-277050610 and out.
